modules/exploits/multi/http/php_fpm_rce.rb

```diff
@@ -301,12 +301,13 @@ def send_attack_chain
   end
 
   def send_payload
+    encoded_payload = framework.encoders.create('php/base64').encode(payload.raw, nil, nil)
```

Comment:
I don't think this would be quite the right fix - maybe this is a regression from the recent PHP changes? 🤔 

https://github.com/rapid7/metasploit-framework/pull/20160

cc @zeroSteiner

Reply:
Ok, I agree, the default exploit just mentioned "payload.encoded" but since there was only 1 PHP encoder at the time that was probably not the proper way.
A better solution would indeed bo to have an (advanced) exploit option to specify the encoder and to have it default to Base64 instead of Hex.

Reply:
This is a regression but not a recent one. I bisected it back to determine it was broken in 24750deab37a56cf1de4935f52f94350fe09fa3c from #19420.